charts/trino/values.yaml

```diff
@@ -96,6 +96,67 @@ accessControl: {}
   #       ]
   #     }
 
+resourceGroups: {}
+   # Rules file is mounted to /etc/trino/resource-groups/resource-groups.json
```

Comment:
nit: can you align the commented out lines to have the same indentation?

Reply:
done